### PR TITLE
restore function subtract; fixes #1

### LINKED ISSUE #1: Test fails
There is a failing test in 08ec2018a

### PR DESCRIPTION
**Pull request by CarolinaRobustini**

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -11,5 +11,5 @@ def subtract(a, b):
     return a + b  # <--- fix this in step 8
```
Comment (vezeli) [inline example.py]: Change here.

File: example.py [hunk 2]
```diff
@@ -11,5 +11,5 @@ def subtract(a, b):
...
 
 
-def test_subtract():
-    assert subtract(2, 3) == -1
+#def test_subtract():
```
Comment (vezeli) [inline example.py]: Uncomment the tests to make sure it works.

File: example.py (hunk 1 above)
Comment (CarolinaRobustini) [reply example.py]: Done it!

File: example.py (hunk 2 above)
Comment (CarolinaRobustini) [reply example.py]: ok!

**Comment by vezeli** [comment]
Looks good :+1:

**Pull request merged**